dashboard/app/models/pd/application/application_base.rb

```diff
@@ -66,6 +66,8 @@ class ApplicationBase < ApplicationRecord
     validates_inclusion_of :application_year, in: APPLICATION_YEARS
 
     # An application either has an "incomplete" or "unreviewed" state when created.
+    # The applied_at field gets set when the status becomes 'unreviewed' for the first time
```

Comment:
I originally took out the `column_exists` check for this update, but the migration didn't go into yesterday's DTP. We can either (a) wait for it to be deployed and remove this check, or (I think), (b) keep this check and merge it in now. @davidsbailey do you have a strong preference either way?

Reply:
(See line below for the actual `column_exists` check ... didn't want to continue the novel from below.

Reply:
ok, so I may have gone a bit overboard in suggesting that the migration PR _ships_ before this PR is merged. the most important thing is that the migration PR is _merged_ first (already done) and that code changes are all extracted into a separate PR (this PR, already done). so I think this is good to merge without the `column_exists?` checks. 

the times that you _would_ want to wait for the migration PR to _ship_ before the next PR would be if you planned to manually inspect the production database to make sure it had worked as expected, which does not seem necessary in this case because it is a simple column addition.

Reply:
to clarify, in production, the migration will have completed before this code becomes live in the dashboard server, due to the order that things happen in our deployment pipeline.

Reply:
Ahh, I see! So if the migration PR is merged before this one, we're guaranteed that the migration happens before this? Didn't know we could rely on that! Great.

Reply:
Excellent, got it. I assumed the PR from a dtp_candidate like https://github.com/code-dot-org/code-dot-org/commit/440f8e8a309be0d949a1ee7ec9957f8d86051d73 can't verify order ... how does this work? I can also ask infra someday

Reply:
in any of our deploys, migrations happen before dashboard is updated:
![Screen Shot 2022-03-18 at 1 12 33 PM](https://user-images.githubusercontent.com/8001765/159076902-fe15dbbb-aebe-44e1-9393-108955133cf5.png)
 
so, it's not that the migration code gets to production first. the possibilities are that the migration gets to prod first, or in the same DTP as, the code change. if they are in the same DTP, then the deploy process ordering shown above ensures that migrations take effect before code changes. please feel free to ping me for more info here

Reply:
Ah I see––thanks!